src/Tests/Eshopworld.Messaging.Tests/ServiceBusFluentExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Eshopworld.Messaging.Tests
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Microsoft.Azure.Management.ServiceBus.Fluent;
+
+    /// <summary>
+    /// Contains extension methods for <see cref="Microsoft.Azure.Management.Fluent"/> around ServiceBus.
+    /// </summary>
+    public static class ServiceBusFluentExtensions
+    {
+        /// <summary>
+        /// Scorches the entire service bus namespace.
+        /// Currently this wipes out all queues and topics. This is used mostly by integration tests, to guarantee that
+        /// both queue and topic creation processes are in place and working as intended.
+        /// </summary>
+        /// <param name="sbNamespace">The <see cref="IServiceBusNamespace"/> that we are scorching.</param>
+        /// <returns>The async <see cref="Task"/> wrapper.</returns>
+        public static async Task ScorchNamespace(this IServiceBusNamespace sbNamespace)
```

Comment:
I would move this to the non test extension class, this could have valid use outside of IntTest